pennylane_lightning/core/src/simulators/lightning_qubit/gates/cpu_kernels/avx_common/TwoQubitGateHelper.hpp

```diff
@@ -353,7 +353,9 @@ class TwoQubitGateWithoutParamHelper {
      */
     auto operator()(std::complex<Precision> *arr, const size_t num_qubits,
                     const std::vector<size_t> &wires, bool inverse) const
-        -> ReturnType requires SymmetricTwoQubitGateWithoutParam<AVXImpl> {
+        -> ReturnType
+        requires SymmetricTwoQubitGateWithoutParam<AVXImpl>
+    {
```

Comment:
Hmmm, these shouldn't be on this line, but on the previous one.  Dame with everywhere below. If this isn't movable, I think we can accept for now and fix later, but I wonder why it isn't uniform across functions

Reply:
Should we just turn off clang-format for those lines?